### ISSUE TITLE
Segmentation Fault after scenes LibLoaded

### ISSUE THREAD

**Issue by vlad0337187** [UPBGE/upbge#928]
Just a segmentation fault, don't know it's reason.
Both scenes separately worked very well, I walked across them many times.

But after two scenes loaded, I receive segmentation fault.
Crash log: [run.crash.txt](https://github.com/UPBGE/blender/files/2563953/run.crash.txt)

Video with reproducing steps: https://www.youtube.com/watch?v=8bDOm4CHSdE

Don't know, maybe I do something wrong.
Project:  https://bitbucket.org/new_novel/oasis/src
Btw, feel free to join project =)

System:  Linux Mint 19 Cinnamon
Kernel:  4.15.0-38-generic
CPU:  Intel© Core™ i5-5200U CPU @ 2.20GHz × 2
Memory:  15.6 GiB
Graphics:  Nvidia Geforce GT 840m
UPBGE:  0.2.4

**Comment by lordloki** [comment]
It was impossible to crash in debug mode (I had to move the player with 0.5fps). I had a lot of assert in RAS_Mesh::GetPolygon()
![image](https://user-images.githubusercontent.com/1015605/48261349-0abbd080-e41f-11e8-9d92-b4e71a0a0b2f.png)

In Release mode it crashes after a while. I think this is not related to LibLoad.

**Comment by vlad1777d** [comment]
@lordloki , hm...
When I link character to scene, there are no crashes. 
I walk with it on scene a lot of time to test physics and just look into it.

Hm... Can I help with it somehow ?
I tried to compile UPBGE recently, but I only ran install dependencies script, it compiled near hour, than said me: Lib A is absent.
I installed it manually.  Than it said Lib B absent and so on.

Now I'm stacked with some lib, but I rebooted Laptop, so don't remember  it's name.
Hope, soon I could compile it =)

What should I do ?
Compile UPBGE in debug mode, launch that scene and write here traceback ?
